### DISCUSSION TITLE
Helm Deployment to EKS Stopped Working

### DISCUSSION THREAD

**Discussion by osiris-dev** [community/community#51623] [Actions]
### Select Topic Area

Question

### Body

Hi team!

The helm deploy step of my workflow stopped working about a week ago. Up until then it has always worked great. It now suddenly complains about credentials 

> Error: INSTALLATION FAILED: Kubernetes cluster unreachable: the server has asked for the client to provide credentials

Can someone please help me out? Is anyone aware of any changes made in this space that could have resulted in that breakage? The step:

>   Deploy-Backend-k8s:
    runs-on: ubuntu-latest
    if: github.ref == 'refs/heads/master'
    steps:
    - name: Configure AWS credentials
      uses: aws-actions/configure-aws-credentials@v1
      with:
        aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
        aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
        aws-region: us-west-2
    - name: Login to Amazon ECR
      id: login-ecr
      uses: aws-actions/amazon-ecr-login@v1
    - name: Deploy to EKS
      env:
        HELM_EXPERIMENTAL_OCI: 1
      run: |
        aws eks --region us-east-1 update-kubeconfig --name x
        kubectl config set-context --current --namespace=x
        helm pull --untar oci://*.dkr.ecr.us-west-2.amazonaws.com/x --version 1.0.0
        helm upgrade --install --recreate-pods --wait -f x/values.yaml x ./x

**Comment by miparnisari** [comment]
Isn't the `docker/login-action@v2` mising the `registry` field? E.g.

```yaml
      - name: Login to AWS ECR
        uses: docker/login-action@v2
        with:
          registry: ${{ secrets.ECR_REGISTRY_URI }}
```
